src/main/java/de/srendi/advancedperipherals/common/util/LuaConverter.java

```diff
@@ -30,15 +37,17 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
 import java.util.function.Supplier;
 import java.util.stream.Stream;
 
 public class LuaConverter {
 
-    public static Map<String, Object> entityToLua(Entity entity, boolean detailed) {
+    public static Map<String, Object> entityToLua(Entity entity) {
         Map<String, Object> data = new HashMap<>();
-        data.put("id", entity.getId());
-        data.put("uuid", entity.getStringUUID());
+        data.put("teamName", ObjectUtil.nullableValue(entity.getTeam(), Team::getName));
```

Comment:
teamName, inFluid, dimension, isGlowing, isUnderWater, isInLava, and isInWall can counted as detailed information